.vscode/launch.json

```diff
@@ -61,27 +61,6 @@
                 "MOCHA_grep": "", // RegExp of tests to run (empty means all)
             }
         },
-        {
-            "name": "Launch Tests (unit)",
```

Comment:
This launch config isn't really needed anymore. All our tests are very fast, and I have no intention of adding back slow ones. :smile: